EnTT really work on windows ?

Hey using vcpkg + entt 2.6.1

```cpp
  entity_registry_.each([this](auto entity) {
                this->entity_registry_.destroy(entity);
            });
````

this snippet doesn't compile with the following error
  ecs-test.cpp

c:\tools\vcpkg\installed\x64-windows\include\entt\entity\view.hpp(567): warning C4003: not enough actual parameters for macro 'min' [C:\projects\shiva\build\tests\ecs-test\ecs-test.vcxproj]
c:\tools\vcpkg\installed\x64-windows\include\entt\entity\view.hpp(625): warning C4003: not enough actual parameters for macro 'min' [C:\projects\shiva\build\tests\ecs-test\ecs-test.vcxproj]
c:\tools\vcpkg\installed\x64-windows\include\entt\entity\view.hpp(633): warning C4003: not enough actual parameters for macro 'max' [C:\projects\shiva\build\tests\ecs-test\ecs-test.vcxproj]
c:\tools\vcpkg\installed\x64-windows\include\entt\entity\view.hpp(756): warning C4003: not enough actual parameters for macro 'min' [C:\projects\shiva\build\tests\ecs-test\ecs-test.vcxproj]
c:\tools\vcpkg\installed\x64-windows\include\entt\entity\view.hpp(552): error C2589: '(': illegal token on right side of '::' [C:\projects\shiva\build\tests\ecs-test\ecs-test.vcxproj]
  c:\tools\vcpkg\installed\x64-windows\include\entt\entity\view.hpp(901): note: see reference to class template instantiation 'entt::View<Entity,Component...>' being compiled
c:\tools\vcpkg\installed\x64-windows\include\entt\entity\view.hpp(567): error C2589: '(': illegal token on right side of '::' [C:\projects\shiva\build\tests\ecs-test\ecs-test.vcxproj]
c:\tools\vcpkg\installed\x64-windows\include\entt\entity\view.hpp(625): error C2589: '(': illegal token on right side of '::' [C:\projects\shiva\build\tests\ecs-test\ecs-test.vcxproj]
c:\tools\vcpkg\installed\x64-windows\include\entt\entity\view.hpp(633): error C2589: '(': illegal token on right side of '::' [C:\projects\shiva\build\tests\ecs-test\ecs-test.vcxproj]
c:\tools\vcpkg\installed\x64-windows\include\entt\entity\view.hpp(756): error C2589: '(': illegal token on right side of '::' [C:\projects\shiva\build\tests\ecs-test\ecs-test.vcxproj]

It's does with /W4 msvc flags
`c:\tools\vcpkg\installed\x64-windows\include\entt\entity\view.hpp(567): warning C4003: not enough actual parameters for macro 'min' [C:\projects\shiva\build\tests\ecs-test\ecs-test.vcxproj]`

--- a/AUTHORS
+++ b/AUTHORS
@@ -11,6 +11,7 @@ ceeac
 ColinH
 corystegel
 Croydon
+cugone
 dbacchet
 dBagrat
 djarek

--- a/src/entt/entity/view.hpp
+++ b/src/entt/entity/view.hpp
@@ -240,7 +240,7 @@ class basic_view<Entity, exclude_t<Exclude...>, Component...> {
     {}
 
     [[nodiscard]] const sparse_set<Entity> & candidate() const ENTT_NOEXCEPT {
-        return *std::min({ static_cast<const sparse_set<Entity> *>(std::get<pool_type<Component> *>(pools))... }, [](const auto *lhs, const auto *rhs) {
+        return *(std::min)({ static_cast<const sparse_set<Entity> *>(std::get<pool_type<Component> *>(pools))... }, [](const auto *lhs, const auto *rhs) {
             return lhs->size() < rhs->size();
         });
     }
@@ -320,7 +320,7 @@ public:
      * @return Estimated number of entities iterated by the view.
      */
     [[nodiscard]] size_type size() const ENTT_NOEXCEPT {
-        return std::min({ std::get<pool_type<Component> *>(pools)->size()... });
+        return (std::min)({ std::get<pool_type<Component> *>(pools)->size()... });
     }
 
     /**

